llvm/lib/Target/AArch64/AArch64LoadStoreOptimizer.cpp

```diff
@@ -2782,6 +2815,9 @@ bool AArch64LoadStoreOpt::tryToMergeIndexLdSt(MachineBasicBlock::iterator &MBBI,
 bool AArch64LoadStoreOpt::optimizeBlock(MachineBasicBlock &MBB,
                                         bool EnableNarrowZeroStOpt) {
   AArch64FunctionInfo &AFI = *MBB.getParent()->getInfo<AArch64FunctionInfo>();
+  bool const CanPairFillSpill = EnableSVEFillSpillPairing &&
+                                Subtarget->isSVEorStreamingSVEAvailable() &&
+                                Subtarget->getSVEVectorSizeInBits() == 128;
```

Comment:
I believe the transformation should be disable for big-endian? because SVE LDR performs 8-bit accesses but SIMD LDP accesses are 128-bit.

Reply:
You're right, sorry. I've fixed this now.